doc/reference/devices_nic.md

```diff
@@ -442,6 +442,7 @@ Key                     | Type    | Default           | Description
 `limits.egress`         | string  | -                 | I/O limit in bit/s for outgoing traffic (various suffixes supported, see {ref}`instances-limit-units`)
 `limits.ingress`        | string  | -                 | I/O limit in bit/s for incoming traffic (various suffixes supported, see {ref}`instances-limit-units`)
 `limits.max`            | string  | -                 | I/O limit in bit/s for both incoming and outgoing traffic (same as setting both `limits.ingress` and `limits.egress`)
+`limits.priority`       | integer | -                 | `skb->priority` for outgoing traffic
```

Comment:
I think we will need to clarify what this config item does in more accessible style, as I doubt many of our users will know what "skb->priority" is. Instead can you expand this to explain what it does as an end result, and whether the priority increases with a higher or lower number? We should also include the range of allowed integers here.

Reply:
@mihalicyn this still needs addressing.

Reply:
I've updated description. Thanks!